OWNERS_ALIASES

```diff
@@ -24,9 +24,9 @@ aliases:
     - thejoycekung # Release Manager Associate
     - wilsonehusin # Release Manager Associate
   release-team:
+    - cici37 # subproject owner / 1.25 Release Team Lead
     - cpanato # subproject owner
     - jameslaverack # subproject owner / 1.24 Release Team Lead
-    - jeremyrickard # subproject owner / 1.20 Release Team Lead
```

Comment:
@jeremyrickard , do you stay on this list as Tech Lead?

Reply:
Yeah, I should be a subproject owner

Reply:
Updated. Thanks!